infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServer.java

```diff
@@ -161,6 +161,17 @@ public SequencerServer(ServerContext serverContext) {
         setUpTimerNameCache();
     }
 
+    @Override
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4651438)